thoth/storages/graph/postgres.py

```diff
@@ -4123,31 +4150,33 @@ def _system_symbols_analysis_result(
         for library, symbols in document["result"]["system-symbols"].items():
             for symbol in symbols:
                 versioned_symbol, _ = VersionedSymbol.get_or_create(
-                    self._session,
+                    session,
                     library_name=library,
                     symbol=symbol,
                 )
                 if is_external:
                     HasSymbol.get_or_create(
-                        self._session,
+                        session,
                         external_software_environment_id=software_environment.id,
                         versioned_symbol_id=versioned_symbol.id
                     )
                 else:
                     HasSymbol.get_or_create(
-                        self._session,
+                        session,
                         software_environment_id=software_environment.id,
                         versioned_symbol_id=versioned_symbol.id
                     )
 
                 DetectedSymbol.get_or_create(
-                    self._session,
+                    session,
                     package_extract_run_id=package_extract_run.id,
                     versioned_symbol_id=versioned_symbol.id
                 )
 
+    @staticmethod
```

Comment:
`_python_sync_analysis_result` is not a staticmethod :)

Reply:
Nice! Fixed in 04d383f